s3.go

```diff
@@ -51,6 +51,16 @@ func (s S3) Store(results []Result) error {
 	return err
 }
 
+// Fetch returns a checkup file -- Not implemented
+func (s S3) Fetch(name string) ([]Result, error) {
+	return nil, fmt.Errorf("S3.Fetch is not implemented")
+}
+
+// GetIndex returns the checkup index -- Not implemented
+func (s S3) GetIndex() (map[string]int64, error) {
+	return nil, fmt.Errorf("S3.GetIndex is not implemented")
+}
```

Comment:
If we split these two methods out of the Storage interface, these two methods can be removed